e2e-tests/cypress/tests/integration/channels/archived_channel/post_menu_spec.ts

```diff
@@ -26,7 +26,7 @@ describe('Archived channels', () => {
 
     it('MM-T1721 Archive channel posts menu should have copy link and reply options', () => {
         // # Click to add a channel description
-        cy.findByRoleExtended('button', {name: 'Add a channel header'}).should('be.visible').click();
+        cy.findByRoleExtended('button', {name: 'Set header'}).should('be.visible').click();
```

Comment:
This still failed.

Reply:
<img width="509" alt="image" src="https://github.com/mattermost/mattermost/assets/15979783/2375c4d6-f5b7-4c1d-8bf8-643eca4145e6">